packages/squidex/src/entities/crn/user.ts

```diff
@@ -16,10 +16,12 @@ import { GraphqlTeam } from './team';
 
 export type UserTeamConnection<T = string> = T extends string
   ? {
+      status: string;
       role: TeamRole;
       id: T[];
     }
   : {
+      status: string;
       role: TeamRole;
       id: T[];
     };
```

Comment:
I think it's weird that the ternary returns the same thing if the condition is true or false, I took a look at the code that introduced this ternary (https://github.com/yldio/asap-hub/pull/632/files)

<img width="537" alt="Screenshot 2023-01-11 at 09 36 37" src="https://user-images.githubusercontent.com/16595804/211808002-8a62598b-8413-45a4-a244-61a3c759c2d1.png">

and the returning value was different when the condition was true or false, but now it does not make sense the ternary, it could be just

```
export type UserTeamConnection<T = string> = {
  status: 'Active' | 'Inactive';
  role: TeamRole;
  id: T[];
};
```

right?